community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/planner/Planner.scala

```diff
@@ -95,15 +96,18 @@ object Planner {
 
   def rewriteStatement(namespacer: Namespacer, statement: Statement): Statement = {
     val rewriter = RewriterStepSequencer.newDefault("Planner")(
+      // ADD HERE ALL THE CONDITION ENABLED IN ASTRewriter
```

Comment:
Instead of having this comment, how about having a list of conditions in a singleton somewhere in ASTRewriter and including that here?